www/docs/en/dev/cordova/events/events.md

```diff
@@ -323,6 +323,16 @@ function onBackKeyDown() {
 }
 ```
 
+### Windows Quirks
+
+Throw an error in a `backbutton` callback to force the default behavior:
+
+```javascript
+document.addEventListener('backbutton', function (evt) {
+    throw "Exit"; // This will suspend the app
```

Comment:
I believe the goal of the overriding is to implement a custom navigation logic as back button closes the app by default on Windows as opposed to its purpose - go back or close if there is no navigational history.
So the only way to control this now is an option to throw an exception in handler so that the app close will happen.